Sources/MongoSwift/MongoCollection.swift

```diff
@@ -133,4 +133,100 @@ public class MongoCollection<T: Codable> {
 
         return try body(collection)
     }
+
+    /**
+     * Starts a `ChangeStream` on a collection. By default, the type `CollectionType` is associated with the
+     * `fullDocument` field in `ChangeStreamDocument` emitted by the returned `ChangeStream`.
+     * - Parameters:
+     *   - pipeline: An array of aggregation pipeline stages to apply to the events returned by the change stream.
+     *   - options: An optional `ChangeStreamOptions` to use when constructing the `ChangeStream`.
```

Comment:
nit: for the other two param descriptions you say "change stream" vs `ChangeStream`, let's be consistent and just say "change stream" for all of them?